### PR TITLE
26 react redux

### PR DESCRIPTION
**Pull request by furaiev**

### CODE REVIEW

File: src/pages/Pages.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
 import { Authorized } from '../Pages/Authorized';
 import { NonAuthorized } from '../Pages/NonAuthorized';
```
Comment (iliatcymbal) [inline src/pages/Pages.js]: 1) where is ../Pages folder?
2) why don't you use _ import { Authorized } from './Authorized';_ ?

**Comment by furaiev** [comment]
@iliatcymbal 
Interesting thing - I've cloned the repo into the new folder and it works. Seems like windows isn't so demanded as your OS.
I've corrected import paths, please try one more time to start the project.

**Comment by iliatcymbal** [comment]
Yes, probably your OS ignore letter case in path name

**Comment by iliatcymbal** [comment]
@furaiev 
You shouldn't request info for non-authenticated users
http://joxi.ru/LmGDl8khe7Kky2
http://joxi.ru/Dr8ElvQH4wjbem

Fix this and merge to master

**Pull request merged**